.yarnrc.yml

```
@@ -0,0 +1,7 @@
+approvedGitRepositories:
+  - "**"
+enableScripts: false
+nodeLinker: node-modules
+npmMinimalAgeGate: 7d
+yarnPath: .yarn/releases/yarn-4.14.1.cjs
+npmPreapprovedPackages: [ "@visitscotland/component-library" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)